schema/NATIONAL_PARTY_EARMARK_MEMOS.json

```diff
@@ -0,0 +1,784 @@
+                    }
+                },
+                "required": [
+                    "entity_type",
+                    "contribution_aggregate"
+                ]
+            },
+            "then": {
+                "required": [
+                    "contributor_employer",
+                    "contributor_occupation"
+                ],
+                "properties": {
+                    "contributor_employer": {
+                        "type": "string"
+                    },
+                    "contributor_occupation": {
+                        "type": "string"
+                    }
+                }
+            }
+        },
+        {
+            "if": {
+                "properties": {
+                    "entity_type": {
+                        "const": "COM"
+                    }
+                },
+                "required": [
+                    "entity_type"
+                ]
+            },
+            "then": {
+                "required": [
+                    "contributor_organization_name",
+                    "donor_committee_fec_id",
+                    "donor_committee_name"
+                ],
+                "properties": {
+                    "contributor_organization_name": {
+                        "type": "string"
+                    },
+                    "donor_committee_fec_id": {
+                        "type": "string"
+                    },
+                    "donor_committee_name": {
+                        "type": "string"
+                    }
+                }
+            }
+        },
+        {
+            "if": {
+                "properties": {
+                    "entity_type": {
+                        "const": "IND"
+                    }
+                },
+                "required": [
+                    "entity_type"
+                ]
+            },
+            "then": {
+                "required": [
+                    "contributor_first_name",
+                    "contributor_last_name"
+                ],
+                "properties": {
+                    "contributor_first_name": {
+                        "type": "string"
+                    },
+                    "contributor_last_name": {
+                        "type": "string"
+                    }
+                }
+            }
+        },
+        {
+            "if": {
+                "properties": {
+                    "transaction_type_identifier": {
+                        "const": "EARMARK_MEMO_RECOUNT_ACCOUNT"
+                    }
+                },
+                "required": [
+                    "transaction_type_identifier"
+                ]
+            },
+            "then": {
+                "properties": {
+                    "aggregation_group": {
+                        "const": "NATIONAL_PARTY_RECOUNT_ACCOUNT"
+                    }
+                },
+                "required": [
+                    "aggregation_group"
+                ]
```

Comment:
This required is not necessary and can be removed because aggregation_group is already required in the required list at the top of the file